Source/svg/SVGParser.swift

```diff
@@ -595,7 +595,7 @@ open class SVGParser {
         let red = CGFloat((rgbValue >> 16) & 0xff)
         let green = CGFloat((rgbValue >> 08) & 0xff)
         let blue = CGFloat((rgbValue >> 00) & 0xff)
-
+        
```

Comment:
Looks like you don't use SwiftLint. Please install it to make sure there is no formatting issues.